src/components/_common/icons/BurgerIcon.tsx

```diff
@@ -0,0 +1,24 @@
+type IconProps = {
+    color: string;
+    fontSize: number;
+  };
+  
+export const BurgerIcon = ({ color, fontSize }: IconProps) => (
+
+<link>
```

Comment:
Link is not a part of icon. Exlude it. Only SVG components in Icons

Reply:
Done